### ISSUE TITLE
p-paginator: not accessible

### ISSUE THREAD

**Issue by kyjus25** [primefaces/primeng#12684]
### Describe the bug

None of the paginator button elements have aria labels on them which fails the Lighthouse report since buttons without inner text is read only as "button". It is impossible to have a 100% Lighthouse score for accessibility with a p-paginator on the page.

<img width="949" alt="Screenshot 2023-02-27 at 4 29 03 PM" src="https://user-images.githubusercontent.com/6710794/221700491-71843592-f8e1-4e80-b0c8-4fb21b6dd9b9.png">

<img width="660" alt="Screenshot 2023-02-27 at 4 30 39 PM" src="https://user-images.githubusercontent.com/6710794/221700734-fa676b5c-7fcf-44ae-a823-1670399dabdb.png">

### Environment

https://primeng.org/paginator

### Reproducer

https://primeng.org/paginator

### Angular version

15.2.0

### PrimeNG version

15.2.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

N/A

### Browser(s)

Chrome

### Steps to reproduce the behavior

1. Navigate to https://primeng.org/paginator in Chrome
2. Right-click and choose "Inspect"
3. Go to the "Lighthouse" tab
4. Generate a report with at least the "Accessibility" category
5. Scroll to the Accessibility area under "Buttons do not have an accessible name"

### Expected behavior

p-paginator passes the Lighthouse test

**Comment by Tilesto** [comment]
Have the same problems. +1
![Uploading image.png…]()

**Comment by taramason** [comment]
Also seeing this issue. I saw PR from 2019 that seems to add the appropriate aria-labels, but they are definitely not showing up on the components when used. Using this library for professional, accessible websites has proven to be incredibly challenging and not reliable.

**Comment by cetincakiroglu** [comment]
Hi,

Thanks a lot for reporting the issue we'll address it in upcoming release!
